### ISSUE TITLE
Date/time input field with date picker fails accessibility due to label issue

### ISSUE THREAD

**Issue by jamesmacwhite** [verbb/formie#1186]
### Describe the bug

Using a date/time field with a date picker, but the visible input field fails accessibility scans because the label is attached to the hidden input field and not the interactive text field.

The hidden input field doesn't actually need a label attached it in this case.

Example:

The label on a date/time field on a live form, this is correct.

```html
<label id="fields-formie-form-64372863789adc9f994-apprenticeship-potential-start-label" class="fui-label" for="fields-formie-form-64372863789adc9f994-apprenticeship-potential-start">When could you start an apprenticeship?</label>
```

Hidden input field (currently, the value of `for` on the label is attached to this field with any ID, but as it is hidden this should not be labelled)

```html
<input type="hidden" id="fields-formie-form-64372863789adc9f994-apprenticeship-potential-start" class="fui-input flatpickr-input" name="fields[ApprenticeshipPotentialStart][datetime]" autocomplete="off" data-fui-id="find-my-perfect-job22-apprenticeship-potential-start" data-fui-message="" aria-required="false">
```

Interactive Input field that should have the ID (currently has none):

```html
<input class="fui-input form-control input active" placeholder="" tabindex="0" type="text">
```

https://dequeuniversity.com/rules/axe/4.4/label?application=AxeChrome

For accessibility, the interactive field should receive the ID currently being set on hidden input, in addition other properties like `autocomplete`, `class`, `aria-required` should be on the interactive element, the hidden field shouldn't need more than `type`, `name` and `value` as it's not there to be interacted with, but would be needed for processing the date value.

### Steps to reproduce

1. Create a form with a date/time field
2. Observe the HTML markup of the rendered form. The hidden input field for the date/time value receives the label not the interactive field.

### Form settings

Date/Time field using the date picker.

### Craft CMS version

3.7.61

### Plugin version

1.6.19

### Multi-site?

No

### Additional context

N.A

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by jamesmacwhite** [comment]
Awesome, thank you!

**Comment by engram-design** [comment]
Fixed in [1.6.20](https://github.com/verbb/formie/releases/tag/1.6.20)
